tools/scalar-schema/README.md

```diff
@@ -20,6 +21,12 @@ $ java -jar target/scalar-schema.jar --cosmos -h <YOUR_ACCOUNT_URI> -p <YOUR_ACC
 ```
   - `-r BASE_RESOURCE_UNIT` is an option. You can specify the RU of each database. The maximum RU in tables in the database will be set. If you don't specify RU of tables, the database RU will be set with this option. When you use transaction function, the RU of the coordinator table of Scalar DB is specified by this option. By default, it's 400.
 
+```console
+# For DynamoDB
+$ java -jar target/scalar-schema.jar --dynamo -f schema.json [-r BASE_RESOURCE_UNIT]
```

Comment:
I'm not fully sure what needs to be specified for DynamoDB but I would assume we need a description to specify information such as where to connect and what credentials to use.
Also ideally, it would be more user-friendly if the same options like `-h` and `-p` can be used for similar data.

Reply:
Thank you for your suggestion!
I modified the `get-client` to use AWS credentials.